spk/python311/Makefile

```diff
@@ -0,0 +1,174 @@
+SPK_NAME = python311
+SPK_VERS = 3.11.0
+SPK_VERS_MAJOR_MINOR = $(word 1,$(subst ., ,$(SPK_VERS))).$(word 2,$(subst ., ,$(SPK_VERS)))
+SPK_REV = 1
+SPK_ICON = src/python3.png
+
+DEPENDS  = cross/$(SPK_NAME)
+DEPENDS += cross/pip
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Python Programming Language.
+DESCRIPTION_FRE = Langage de programmation Python.
+DESCRIPTION_SPN = Lenguaje de programación Python.
+STARTABLE = no
+DISPLAY_NAME = Python 3.11
+CHANGELOG = "Release Python 3.11.0"
+
+HOMEPAGE = https://www.python.org
+LICENSE  = PSF
+
+SERVICE_SETUP = src/service-setup.sh
+
+ENV += PYTHON_OPTIMIZE=1
+PYTHON_LIB_DIR = lib/python$(SPK_VERS_MAJOR_MINOR)
+
+POST_STRIP_TARGET = python311_extra_install
+
+SPK_USR_LOCAL_LINKS = bin:bin/python3.10
+
+WHEELS = src/requirements-crossenv.txt src/requirements-pure.txt src/requirements-abi3.txt
+
+# Force building pure-python wheels
+WHEELS_PURE_PYTHON_PACKAGING_ENABLE = 1
+
+##
+## All configurations below are optional and
+## are provided to demonstrate how to build
+## various wheels.  Uncoment to enable.
+##
```

Comment:
Have you tried invoking a build with all the wheels to confirm everything really is functional?
It requires a few trial & errors as some disable archs and other are explicitly know to work up to certain gcc.

Reply:
What's the best way to test? Not to just uncomment all?

Reply:
well the requirement* files have all the comments explaining what each wheels require.  You cannot test everything all at once as some only build on some arch or another.  But you can enable quite a bit in one shot, then docus on the exceptions afterwards.